src/daft-dsl/src/expr.rs

```diff
@@ -85,6 +86,8 @@ pub enum AggExpr {
         func: FunctionExpr,
         inputs: Vec<ExprRef>,
     },
+    ApproxCountDistinctSketch(ExprRef),
```

Comment:
It might make sense to put a parameter in MergeSketch instead. We shouldn't need an expr for every algo of sketch

Reply:
Updated!